pivy-importer/src/main/groovy/com/linkedin/python/importer/distribution/SourceDistPackage.groovy

```diff
@@ -15,149 +15,61 @@
  */
 package com.linkedin.python.importer.distribution
 
-import java.util.zip.ZipFile
-
 import groovy.util.logging.Slf4j
 import org.apache.commons.compress.archivers.tar.TarArchiveInputStream
 import org.apache.commons.compress.compressors.bzip2.BZip2CompressorInputStream
 import org.apache.commons.compress.compressors.gzip.GzipCompressorInputStream
 
 import com.linkedin.python.importer.deps.DependencySubstitution
 import com.linkedin.python.importer.pypi.PypiApiCache
-import com.linkedin.python.importer.pypi.VersionRange
 
 @Slf4j
-class SourceDistPackage {
-
-    private final File packageFile
-    private final PypiApiCache pypiApiCache
-    private final DependencySubstitution dependencySubstitution
-    private final boolean latestVersions
-    private final boolean allowPreReleases
-
-    SourceDistPackage(
-            File packageFile,
-            PypiApiCache pypiApiCache,
-            DependencySubstitution dependencySubstitution,
-            boolean latestVersions,
-            boolean allowPreReleases) {
-        this.dependencySubstitution = dependencySubstitution
-        this.pypiApiCache = pypiApiCache
-        this.packageFile = packageFile
-        this.latestVersions = latestVersions
-        this.allowPreReleases = allowPreReleases
+class SourceDistPackage extends PythonPackage {
+
+    SourceDistPackage(File packageFile,
+                      PypiApiCache pypiApiCache,
+                      DependencySubstitution dependencySubstitution,
+                      boolean latestVersions,
+                      boolean allowPreReleases,
+                      boolean lenient) {
+
+        super(packageFile, pypiApiCache, dependencySubstitution, latestVersions, allowPreReleases, lenient)
     }
 
+    @Override
     Map<String, List<String>> getDependencies() {
         return parseRequiresText(getRequiresTextFile())
     }
 
     @SuppressWarnings("ParameterReassignment")
     private Map<String, List<String>> parseRequiresText(String requires) {
-        def dependencies = [:]
+        def dependenciesMap = [:]
```

Comment:
I want the name to be more explanatory, normally dependencies should be a list or set, but this is a Map in this case, so I decide to add "Map" to it to make it more easy to understand.

Reply:
Ok the convention I've heard in general is that it's not necessary to make the type be part of the variable name because that information is easily accessible through the IDE/declaration.

Reply:
I want to differentiate dependencies map from dependencies list, since I use "dependencies" for list, and "dependenciesMap" for map in other places, so I made changes here to use the same way, otherwise, it's easy to get confused.

Reply:
I understand the motivation, but having a dependencies list and dependencies map in the same scope is already confusing.  Also, gratuitous renaming (or reformatting) of things just make code diffs harder to read, and changes harder to merge.  There is a *lot* of value in not touching things that don't need to be touched.

Reply:
They are not in the same scope, "dependencies" sounds more like a List to me, if we make "dependencies" denote both Map and List, then this is confusing in my opinion.